slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -434,22 +434,31 @@ class SlideToActView @JvmOverloads constructor(
         val widthSize = MeasureSpec.getSize(widthMeasureSpec)
         val width: Int
 
+        val heightMode = MeasureSpec.getMode(heightMeasureSpec)
+        val heightSize = MeasureSpec.getSize(heightMeasureSpec)
+        val height: Int
+
         width = when (widthMode) {
             MeasureSpec.EXACTLY -> widthSize
             MeasureSpec.AT_MOST -> Math.min(mDesiredSliderWidth, widthSize)
             MeasureSpec.UNSPECIFIED -> mDesiredSliderWidth
             else -> mDesiredSliderWidth
         }
-        setMeasuredDimension(width, mDesiredSliderHeight)
+
+        height = when (heightMode) {
+            MeasureSpec.EXACTLY -> heightSize
+            MeasureSpec.AT_MOST -> Math.min(mDesiredSliderHeight, heightSize)
+            MeasureSpec.UNSPECIFIED -> mDesiredSliderHeight
+            else -> mDesiredSliderHeight
+        }
+
+        setMeasuredDimension(width, height)
     }
 
     override fun onSizeChanged(w: Int, h: Int, oldw: Int, oldh: Int) {
         mAreaWidth = w
         mAreaHeight = h
-        if (mBorderRadius == -1) {
```

Comment:
Why have you removed this?

Reply:
It has been removed in order to allow mBorderRadius to be re-calculated when the size changes.
Without this, if the available space increases in height (as depicted in Picture 1 to Picture 2) the round slider buttons are not round anymore (See Picture 2).

Reply:
Great thanks for clarifying 👍

Reply:
You are welcome